docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,224 @@
+
+```code
+$ tsh db ls
+Name                                               Description Allowed Users   Labels  Connect
+-----------------------------------------------    ----------- --------------- ------- -------------------------
+myelastic                                                      [alice elastic] env=dev
+mysql-server1 (user: alice, db: teleport_example)              [alice elastic] env=dev tsh db connect mysql-server1
+```
+
+To connect to a database server, through `tsh`, you'll need a local client for
+that database. For example, to connect to a MySQL or MariaDB database, you'll
+need the [MySQL CLI](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) client:
+
+```code
+tsh db connect --db-user=alice --db-name=teleport_example mysql-server1 
+```
+
+In this example, `teleport_example` is a pre-existing database on the MySQL server:
+
+```sql
+SHOW DATABASES;
++--------------------+
+| Database           |
++--------------------+
+| information_schema |
+| mysql              |
+| performance_schema |
+| teleport_example   |
++--------------------+
+4 rows in set (0.09 sec)
+```
+
+For other connection types, or to connect a graphic database client, you can create a local tunnel to point your software to:
+
+```code
+$ tsh proxy db myelastic --db-user=alice --tunnel
+Started authenticated tunnel for the Elasticsearch database "myelastic" in cluster "example.come" on 192.0.2.58:52669.
+
+Use one of the following commands to connect to the database:
+
+  * interactive SQL connection:
+
+  $ elasticsearch-sql-cli http://localhost:52669/
+
+  * run single request with curl:
+
+  $ curl http://localhost:52669/
+
+Database certificate renewed: valid until 2022-11-02T06:11:50Z [valid for 9h55m0s]
+```
+
+</TabItem>
+<TabItem label="Teleport Connect">
+
+In the **Databases** tab, select the database you want to access and click
+**CONNECT**. Select or type in a username available to your Teleport user.
+
+Teleport Connect will open a tunneled connection to the database. You can copy the port number and connect with a GUI app at `localhost`, or click **RUN** to initiate a CLI connection inside Teleport Connect:
+
+![Teleport Connect database connection](../img/teleport-connect-mysql.png)
+
+</TabItem>
+</Tabs>
+
+## Desktop Access
+
+Desktop Access is available through the Teleport web UI. In your browser,
+navigate to your Telport cluster (e.g. 
+<ScopedBlock scope={["oss", "enterprise"]}>`https://teleport.example.com`</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>`https://mytennant.teleport.sh`</ScopedBlock>
+). From the menu on the right, select <Icon name="desktop" inline size="sm"/> **Desktops**. Next to the desktop you want to access, click **CONNECT**. Select
+or type in a username available to your Teleport user. Teleport will open a new
+browser tab or window and begin the RDP session. Note that you may need to wait
+a moment for Teleport to log you in as the specified user.
+
+## Next Steps
```

Comment:
```suggestion
## Next steps
```
We're using sentence case in docs headings

Reply:
Sorry, that's a hard habit to break 😄